docs/src/main/paradox/release-notes/2.0.x.md

```diff
@@ -14,6 +14,28 @@ The Alpakka Kafka 2.0 series features
 * built-in @ref:[Akka Discovery](../discovery.md) support
 * More versatile use of Testcontainers in Alpakka Kafka Testkit
```

Comment:
For future, and not related to this PR, but I'd make the bullet points, capitalization or not, consistent.